activity_calendar/views.py

```diff
@@ -396,10 +405,12 @@ def form_invalid(self, form):
 # #######################################
 
 
-class EditActivityMomentView(LoginRequiredMixin, PermissionRequiredMixin, ActivityMixin, FormView):
+class EditActivityMomentView(LoginRequiredMixin, ActivityMixin, UserPassesTestMixin, FormView):
     form_class = ActivityMomentForm
     template_name = "activity_calendar/activity_moment_form_page.html"
-    permission_required = ('activity_calendar.change_activitymoment',)
+
+    def get_test_func(self):
```

Comment:
I don't think this is needed anymore ;)

Reply:
It is, it is used by UserPassesTestMixin to stop access to this particular page if can_edit_activity returns false. This is not default behaviour from the ActivityMixin